<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>passzero</class>
 <widget class="QMainWindow" name="passzero">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pass Zero</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>611</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>260</y>
        <width>711</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Create a new database or open an existing one to continue.</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="btnDelete">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnNewEntry">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>New Entry</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>211</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>65</y>
        <width>74</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
       </font>
      </property>
      <property name="text">
       <string>Item Label:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textLabel">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>60</y>
        <width>351</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>125</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textUsername">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>120</y>
        <width>351</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>185</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textPassword">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>180</y>
        <width>351</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>250</y>
        <width>101</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Secure Notes:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textNote">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>270</y>
        <width>351</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionChange_Master_Key"/>
    <addaction name="actionRotate_Database_keys"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDatabase"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionChange_Master_Key">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Change Master Key</string>
   </property>
  </action>
  <action name="actionRotate_Database_keys">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rotate Database key</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
